src/main/java/de/hysky/skyblocker/skyblock/special/DyeSpecialEffects.java

```diff
@@ -0,0 +1,58 @@
+package de.hysky.skyblocker.skyblock.special;
+
+import java.util.Optional;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.jetbrains.annotations.VisibleForTesting;
+import org.slf4j.Logger;
+
+import com.mojang.logging.LogUtils;
+
+import de.hysky.skyblocker.config.SkyblockerConfigManager;
+import de.hysky.skyblocker.skyblock.itemlist.ItemRepository;
+import de.hysky.skyblocker.utils.Utils;
+import net.fabricmc.fabric.api.client.message.v1.ClientReceiveMessageEvents;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.item.ItemStack;
+import net.minecraft.particle.ParticleTypes;
+import net.minecraft.text.Text;
+
+public class DyeSpecialEffects {
+	private static final Logger LOGGER = LogUtils.getLogger();
```

Comment:
We use `LoggerFactory.getLogger` but whatever.

Reply:
Mojang's logging library calls that for us with the class specified already :)